### ISSUE TITLE
iPad APP Couldn’t Open

### ISSUE THREAD

**Issue by polingsays** [FlutterFlow/flutterflow-issues#2468]
### Has your issue been reported?

- [X] I have searched the existing issues and confirm it has not been reported.
- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

On iPad Pro 11” with M2 chip, when opening the app, it shows a full black screen, and nothing.
This error appears after an update of the FF APP.

### Expected Behavior

Opening the app smoothly, and start the dashboard for building the app.

### Steps to Reproduce

1. Open the app for iPads
2. The situation appears

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

NA / UNABLE TO GET SINCE NOT OPENING

### Context

Unable to build anything on iPads.

### Visual documentation

![IMG_0023](https://uploads.linear.app/3241089d-736a-4422-900b-051e72accdbe/e090e488-0551-4041-9cdf-7cec0528c065/e1c3e328-96a6-4538-94d2-9f0045ec9eb3?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiLzMyNDEwODlkLTczNmEtNDQyMi05MDBiLTA1MWU3MmFjY2RiZS9lMDkwZTQ4OC0wNTUxLTQwNDEtOWNkZi03Y2VjMDUyOGMwNjUvZTFjM2UzMjgtOTZhNi00NTM4LTk0ZDItOWYwMDQ1ZWM5ZWIzIiwiaWF0IjoxNzA5OTAwNTQ0LCJleHAiOjMzMjgwNDYwNTQ0fQ.1F0h4YTq2lSnp_N6OuqyFddL8y6X7YzlZrazFD4h8Xo)

### Additional Info

*No response*

### Environment

```markdown
- FlutterFlow version: latest 4.1.30 iPad
- Platform: iPadOS
- Browser name and version:
- Operating system and version affected: iPadOS
```

**Comment by Pepino-GR** [comment]
I got the same result after update.
iPadOS-Version 17.4
IPadPro

**Comment by clarkie1977** [comment]
I have the same issue. I have deleted the app, restarted my iPad Pro 12.9 M1 and reinstalled FlutterFlow to no avail.

**Comment by sbis04** [comment]
Should be fixed on the latest version, `4.31.1`. We are really sorry for the issue. It's unfortunate that somehow a bad version went through the App Store review and got published, without any warnings.
